cli/internal/fs/hash.go

```diff
@@ -4,10 +4,10 @@ import (
 	"crypto/sha1"
 	"encoding/hex"
 	"fmt"
+	"github.com/vercel/turborepo/cli/internal/xxhash"
```

Comment:
Can you run `gofmt`? I think this is not currently the correct formatting. Or just remove this file from the PR, I don't think the change is relevant.